### PR TITLE
GFM-69 - Change from 5 line addresses to 1 textarea

### PR DESCRIPTION
**Pull request by gxxm**
* Modified address controller
* Change views - address-inside, address-outside, no-postcode, postcode
* Modified summary partial - contact-details
* Modified Steps to use single field
* Modified address-outside fields to use flag ignore-defaults to ignore default formatters

### CODE REVIEW

**Comment by joefitter** [comment]
👍  This LGTM, although we now have a few near-identical steps, these should be cleaned up and made into one step with conditional content etc, but not in the scope of this PR

**Comment by joefitter** [comment]
Happy to merge if you are @JoeChapman

File: apps/gro/fields/address-outside.js [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-  },
-  'address-text-three': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-four': {
-    labelClassName: 'visuallyhidden'
-  },
-  'address-text-five': {
-    labelClassName: 'visuallyhidden'
+    'ignore-defaults': true
```
Comment (JoeChapman) [inline apps/gro/fields/address-outside.js]: what does this do?

File: assets/scss/app.scss [hunk 2]
```diff
@@ -151,3 +151,6 @@ span.postcode {
   margin-right: 0.5em;
 }
 
+#address-text {
+  height: 6em;
```
Comment (JoeChapman) [inline assets/scss/app.scss]: is this the right way to do this, should you use line-height and font-size instead?

File: apps/gro/fields/address-outside.js (hunk 1 above)
Comment (joefitter) [inline apps/gro/fields/address-outside.js]: This stops the default formatters being applied - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/form.js#L19

The param is in formatters/index - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L20

This formatter is applied by default and gets rid of line breaks etc - https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L20

Another hidden feature

Comment (joefitter) [inline apps/gro/fields/address-outside.js]: There is a textarea in BRP that has all whitespace replaced with single spaces too - should prob be updated if you want to preserve line breaks there too

Comment (JoeChapman) [inline apps/gro/fields/address-outside.js]: Sp is `adress-text-five` a textarea?

Comment (joefitter) [inline apps/gro/fields/address-outside.js]: @gxxm this setting will remove all default formatters - we should still use the others `trim` and `hyphens` - pass `formatter: ['trim', 'hyphens']`

File: assets/scss/app.scss (hunk 2 above)
Comment (joefitter) [inline assets/scss/app.scss]: @JoeChapman this is a textarea, should probably be renamed accordingly

Comment (gxxm) [inline assets/scss/app.scss]: Using address-textarea to be more explicit.

File: apps/gro/fields/address-outside.js (hunk 1 above)
Comment (gxxm) [inline apps/gro/fields/address-outside.js]: Added formatter: ['trim', 'hyphens'].

**Comment by JoeChapman** [comment]
@joefitter @ben are you ok with this?

**Comment by joefitter** [comment]
yeah whatever

**Comment by bendiggle** [comment]
LGTM

**Pull request merged**